### PR TITLE
[guides] Review PRODUCTION_RU.md

### PR DESCRIPTION
**Pull request by kirkonru**
## Description

Reviewed and rephrased the PRODUCTION_RU.md file.

## Why do we need it, and what problem does it solve?

Makes documentation easier to read.

### CODE REVIEW

File: docs/site/pages/guides/PRODUCTION_RU.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ lang: ru
 В кластере должно быть три мастер-узла с быстрыми дисками 400+ IOPS.
 {% endalert %}
 
-Всегда используйте три мастер-узла, так как их достаточно для отказоустойчивости. Также это дает возможность безопасно обновлять control plane кластера и сами мастер-узлы. В большем количестве мастер-узлов нет необходимости, а 2 узла (как и любое четное количество) не дают кворума.
+Всегда используйте три мастер-узла — такое количество обеспечит отказоустойчивость и позволит безопасно обновлять control plane кластера и сами мастер-узлы. В большем числе мастер-узлов нет необходимости, а любое их четное количество не дает никаких преимуществ по сравнению с нечетным числом, меньшем на единицу (т.е. с точки зрения отказоустойчивости 2 мастер-узла ничем не отличаются от одного).
```
Comment (kirkonru) [inline docs/site/pages/guides/PRODUCTION_RU.md:69]: @z9r5 please check my passage here :sweat_smile:

File: docs/site/pages/guides/PRODUCTION_RU.md [hunk 2]
```diff
@@ -4,47 +4,47 @@ permalink: ru/guides/production.html
...
+Если указан автоматический выбор версии Kubernetes, Deckhouse может обновить версию Kubernetes в кластере при обновлении релиза Deckhouse (при обновлении минорной версии). Когда версия Kubernetes явно прописана в параметре [kubernetesVersion](/documentation/v1/installing/configuration.html#clusterconfiguration-kubernetesversion), очередное обновление Deckhouse может завершиться неудачей, если окажется, что используемая в кластере версия Kubernetes более не поддерживается.
 
-Решите, стоит ли вам использовать автоматический выбор версии или указать конкретную версию и периодически обновлять ее вручную.
+Решите, стоит ли использовать автоматический выбор версии или указать конкретную версию и периодически обновлять ее вручную.
```
Comment (kirkonru) [inline docs/site/pages/guides/PRODUCTION_RU.md:37]:
```suggestion
Если указана конкретная версия, пользователь сам отвечает за ее периодическое обновление в ручном режиме.
```

File: docs/site/pages/guides/PRODUCTION_RU.md (hunk 1 above)
Comment (z9r5) [inline docs/site/pages/guides/PRODUCTION_RU.md:69]:
```suggestion
Всегда используйте три мастер-узла — такое количество обеспечит отказоустойчивость и позволит безопасно выполнять обновление мастер-узлов. В большем числе мастер-узлов нет необходимости, а два узла (как и любое четное число) не обеспечат наличие кворума.
```

Можно убрать "(как и любое четное число)" с целью упрощения

Comment (kirkonru) [reply docs/site/pages/guides/PRODUCTION_RU.md:69]:
```suggestion
Всегда используйте три мастер-узла — такое количество обеспечит отказоустойчивость и позволит безопасно выполнять обновление мастер-узлов. В большем числе мастер-узлов нет необходимости, а два узла не обеспечат кворума.
```
Тогда предлагаю и "наличие" убрать

File: docs/site/pages/guides/PRODUCTION_RU.md [hunk 3]
```diff
@@ -4,47 +4,47 @@ permalink: ru/guides/production.html
...
+Если указан автоматический выбор версии Kubernetes, Deckhouse может обновить версию Kubernetes в кластере при обновлении релиза Deckhouse (при обновлении минорной версии). Когда версия Kubernetes явно прописана в параметре [kubernetesVersion](/documentation/v1/installing/configuration.html#clusterconfiguration-kubernetesversion), очередное обновление Deckhouse может завершиться неудачей, если окажется, что используемая в кластере версия Kubernetes более не поддерживается.
 
-Решите, стоит ли вам использовать автоматический выбор версии или указать конкретную версию и периодически обновлять ее вручную.
+Решите, стоит ли использовать автоматический выбор версии. Возможно, вам будет удобнее указать конкретную версию и периодически обновлять ее вручную.
```
Comment (z9r5) [inline docs/site/pages/guides/PRODUCTION_RU.md:37]:
```suggestion
```

File: docs/site/pages/guides/PRODUCTION_RU.md [hunk 4]
```diff
@@ -130,21 +129,21 @@ Inlet `HostWithFailover` подходит для кластеров с одни
 Настройте отправку алертов через [внутренний](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-alertmanager) Alertmanager или подключите [внешний](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-внешний-дополнительный-alertmanager).
 {% endalert %}
 
-Мониторинг будет работать сразу после установки Deckhouse, но при работе в Production этого недостаточно. Настройте [встроенный](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-alertmanager) в Deckhouse Alertmanager или [подключите свой](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-внешний-дополнительный-alertmanager) Alertmanager, чтобы получать уведомления об инцидентах.
+Мониторинг будет работать сразу после установки Deckhouse, однако для production этого недостаточно. Чтобы получать уведомления об инцидентах, настройте Alertmanager, [встроенный](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-alertmanager) в Deckhouse, или [подключите свой](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-внешний-дополнительный-alertmanager).
```
Comment (z9r5) [inline docs/site/pages/guides/PRODUCTION_RU.md:132]:
```suggestion
Мониторинг будет работать сразу после установки Deckhouse, однако для production этого недостаточно. Чтобы получать уведомления об инцидентах, настройте [встроенный](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-alertmanager) в Deckhouse Alertmanager, или [подключите свой](/documentation/v1.44/modules/300-prometheus/faq.html#как-добавить-внешний-дополнительный-alertmanager) Alertmanager.
```
@kirkonru

**Pull request merged**